src/esp/bindings_js/bindings_js.cpp

```diff
@@ -204,6 +203,16 @@ EMSCRIPTEN_BINDINGS(habitat_sim_bindings_js) {
       .function("setLocalTransform", &Sensor_setLocalTransform)
       .function("specification", &Sensor::specification);
 
+  em::class_<SensorSuite>("SensorSuite")
+      .smart_ptr<SensorSuite::ptr>("SensorSuite::ptr")
+      .function("add", &SensorSuite::add)
+      .function("merge", &SensorSuite::merge)
+      .function("remove",
+                em::select_overload<void(const Sensor&)>(&SensorSuite::remove))
+      .function("remove", em::select_overload<void(const std::string&)>(
+                              &SensorSuite::remove))
+      .function("clear", &SensorSuite::clear);
+
```

Comment:
A note about SensorSuite's bindings: Since SensorSuite::get now returns a Sensor& instead of Sensor::ptr, I am unable to write a binding for it as doing so would cause an `Allocating an object of abstract class TypeError`, as Sensor as pure virtual functions.

I did some research on this and found that it's possible to implement pure virtual functions in JavaScript, but that seems like not what we want, as derived classes will have their own implementation. https://emscripten.org/docs/porting/connecting_cpp_and_javascript/embind.html#deriving-from-c-classes-in-javascript

The js code still works without this SensorSuite::get method, as much of the logic for SensorSuites happens within other classes. However, this might be something that we want in the future. Does anyone with more experience in emscripten know if it is possible to bind such a method, and can give some tips to do so?